cache_images/fedora_packaging.sh

```diff
@@ -7,35 +7,28 @@
 
 set -e
 
-echo "Updating/Installing repos and packages for $OS_REL_VER"
-
-source $GOSRC/$SCRIPT_BASE/lib.sh
-
-req_env_var GOSRC SCRIPT_BASE BIGTO INSTALL_AUTOMATION_VERSION FEDORA_BASE_IMAGE PRIOR_FEDORA_BASE_IMAGE
+SCRIPT_FILEPATH=$(realpath "$0")
+SCRIPT_DIRPATH=$(dirname "$SCRIPT_FILEPATH")
+REPO_DIRPATH=$(realpath "$SCRIPT_DIRPATH/../")
 
-# Pre-req. to install automation tooing
-$LILTO $SUDO dnf install -y git
+# shellcheck source=./lib.sh
+source "$REPO_DIRPATH/lib.sh"
 
-# Install common automation tooling (i.e. ooe.sh)
-curl --silent --show-error --location \
-     --url "https://raw.githubusercontent.com/containers/automation/master/bin/install_automation.sh" | \
-     $SUDO env INSTALL_PREFIX=/usr/share /bin/bash -s - "$INSTALL_AUTOMATION_VERSION"
-# Reload installed environment right now (happens automatically in a new process)
-source /usr/share/automation/environment
+echo "Updating/Installing repos and packages for $OS_REL_VER"
 
 # Set this to 1 to NOT enable updates-testing repository
 DISABLE_UPDATES_TESTING=${DISABLE_UPDATES_TESTING:0}
 
 # Do not enable updates-testing on the previous Fedora release
 if ((DISABLE_UPDATES_TESTING!=0)); then
```

Comment:
One reason to use positive variable names: it's easier to catch logic bugs like this one.(But the most important reason is that it's a lower cognitive load on maintainers)

Reply:
yep...gotcha and gotcha...thx.